bciers/apps/reporting/src/app/components/additionalInformation/operationEmissionSummary/OperationEmissionSummaryPage.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { actionHandler } from "@bciers/actions";
+import OperationEmissionSummaryForm from "./OperationEmissionSummaryForm";
+import { getAdditionalInformationTaskList } from "@reporting/src/app/components/taskList/3_additionalInformation";
+import { getReportAdditionalData } from "@reporting/src/app/utils/getReportAdditionalData";
+import { NEW_ENTRANT_REGISTRATION_PURPOSE } from "@reporting/src/app/utils/constants";
+
+interface Props {
+  version_id: number; //name of the property from the pageFactory props
+}
+
+const getSummaryData = async (versionId: number) => {
```

Comment:
We've been putting these API calls in `/bciers/libs/actions/src/api/` so they can be reused by other components.
I also suggest renaming to `getOperationEmissionSummaryData()` to avoid confusion with other emissions and summaries